spec_cleaner/rpmpreamble.py

```diff
@@ -445,12 +482,16 @@ def add(self, line):
             self._add_line_value_to('patternprovides', match.group(1), key='Provides')
             return
 
-        elif self.reg.re_provides.match(line) and self.reg.re_patternobsolete.search(line):
+        elif self.reg.re_provides.match(line) and self.reg.re_patternobsolete.search(
+            line
+        ):
             match = self.reg.re_provides.match(line)
             self._add_line_value_to('patternobsoletes', match.group(1), key='Provides')
             return
 
-        elif self.reg.re_obsoletes.match(line) and self.reg.re_patternobsolete.search(line):
+        elif self.reg.re_obsoletes.match(line) and self.reg.re_patternobsolete.search(
+            line
+        ):
```

Comment:
as above, before as one line it was much more readable.

Reply:
Fixed by a higher line length limit (100 chars).